ui/client/components/aiAssistant/AIAssistant.js

```diff
@@ -92,35 +92,36 @@ const AIAssistant = () => {
       const mock = ''; // 'mock-';
 
       const queryResp = await axios.get(
-        `/api/ai-docs/${mock}message?query=${searchPhrase}`
+        `/api/dojo/knowledge/${mock}message?query=${searchPhrase}`
       );
       const response = {};
       response.response = queryResp.data;
 
       // create an object with just the unique filenames as keys
-      const filenamesToDocs = queryResp.data.candidate_paragraphs.reduce((obj, curr) => (
-        { ...obj, [curr.root_name]: null }
+      const document_ids = queryResp.data.candidate_paragraphs.reduce((obj, curr) => (
```

Comment:
This is changed since the API side now returns the actual document IDs instead of filenames. cc @adamgilg